react-app.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module '*.svg' {
```

Comment:
Można by to wrzucić do wspólnego folderu `src/types`.
I sam plik mógłby być też bardziej obrazowy, czego to dotyczy, np `svg.d.ts`